sdk/vaa/payloads.go

```diff
@@ -54,6 +60,19 @@ type (
 		NewContract   Address
 	}
 
+	// BodyTokenBridgeModifyBalance is a governance message to modify accountant balances for the tokenbridge.
+	BodyTokenBridgeModifyBalance struct {
+		Module        string
+		TargetChainID ChainID
+		Sequence      uint64
+		ChainId       ChainID
+		TokenChain    ChainID
+		TokenAddress  Address
+		Kind          uint8
+		Amount        *big.Int
```

Comment:
I believe we're using the [holiman/uint256 package](https://pkg.go.dev/github.com/holiman/uint256) for this elsewhere.

The `uint256.Int` type has a [`Bytes32()`](https://pkg.go.dev/github.com/holiman/uint256#Int.Bytes32) function that should make serialization easier.